docs/docs/tutorials/gettingstarted.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Getting Started with Springboard
+summary: A walkthrough of how to pull down the Springboard project, get it running for the first time in Intellij and 
+start customizing a few pieces of the project.
+authors:
+    - Hilary Fairchild-Carbonell
+date: 2020-06-02
+---
+
+# Getting Started with Springboard
+
+This is a guide to actually starting your first project with Springboard- step by step instructions from pulling the
+project down, getting it running for the first time and a few ways to start customizing it to be your own project.
+
+We use IntelliJ as our IDE so the instructions given below assume you have IntelliJ as well. If you want a tutorial
+using another IDE, let us know and we can add it!
+
+This tutorial intentionally goes step-by-step through common errors or mistakes someone might make while trying to get the project running.
+If you are an experienced developer- some of it might seem obvious and unnecessary to you.
+
+## Pulling the Project form GitHub
+This might be an obvious one- but use the git clone command to pull down the Springboard Project.
+
+```bash
+git clone git@github.com:HappyValleyIO/springboard.git rvroger
```

Comment:
Should we change the name of the project?

Reply:
It was in all the screenshots I took- so I didn't know? Do you want me to redo them? It won't take too long

Reply:
Nah, it'll be fine.